node/heartbeat/monitor_test.go

```diff
@@ -4,86 +4,133 @@ import (
 	"encoding/hex"
 	"encoding/json"
 	"errors"
+	mock2 "github.com/ElrondNetwork/elrond-go/node/heartbeat/mock"
```

Comment:
might rename mock2

Reply:
renamed to mck?

Reply:
Should move the mockTimer.go in node/mock . In this way you won't need this secondary package mck.